web_src/less/_repository.less

```diff
@@ -3066,6 +3066,48 @@ td.blob-excerpt {
   padding-left: 8px;
 }
 
+#diff-container {
+  display: flex;
+}
+#diff-file-boxes {
+  width: 80%;
+}
+
+#diff-file-tree {
+  width: 20%;
+  max-width: 20%;
+  line-height: inherit;
+  position: sticky;
+  padding-top: 0;
+  top: 47px;
+  max-height: calc(100vh - 50px);
+  height: calc(100vh - 50px);
+  overflow-y: auto;
+  overflow-x: auto;
```

Comment:
Remove `overflow-x` and just ellipse the filename if it gets too long. Horizontal scrolling is generally bad UX.

Reply:
Ellipsis is now added 

![image](https://user-images.githubusercontent.com/1135157/188225604-6a8f18b4-e9f1-44d1-9b31-2f8ff3d0dc98.png)